subgraphs/enzyme-core/schema.graphql

```diff
@@ -1542,6 +1542,56 @@ type LidoWithdrawalsRequest @entity {
   claimedTimestamp: Int!
 }
 
+type StakeWiseStakingPosition implements ExternalPosition @entity {
+  id: ID!
+  vault: Vault!
+  type: ExternalPositionType!
+  active: Boolean!
+  stakedEthAmount: BigDecimal!
+  vaultTokens: [StakeWiseVaultToken!]!
+  exitRequests: [StakeWiseStakingExitRequest!]! @derivedFrom(field: "stakeWiseStakingPosition")
+  changes: [ExternalPositionChange!]! @derivedFrom(field: "externalPosition")
+}
+
+type StakeWiseVaultToken @entity {
+  id: ID!
+  createdAt: Int!
```

Comment:
Do we need this field? I see that in `ensureStakeWiseVaultToken` we are setting it to `0`, so intuitively it seems like that could be left out?

Reply:
Had forgotten to set this field. Now implemented.